client/src/lib/i18n/index.ts

```diff
@@ -1,19 +1,21 @@
 import 'intl-pluralrules';
 import i18next from 'i18next';
 import {initReactI18next} from 'react-i18next';
+import {findBestAvailableLanguage} from 'react-native-localize';
```

Comment:
Nice function, any reason we didn't use it before?

Reply:
No idea actually!